content/zh-cn/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -629,7 +629,7 @@ other than `labels` and `annotations`.
 Attempted changes to `name`, `UID` and `namespace` are rejected and fail the request
 which caused the conversion. All other changes are ignored. 
 -->
-#### 被允许的变更
+#### 被允许的变更 {#permissible-mutations}
```

Comment:
Normally, providing an anchor to heading 2 and 3 is a good practice.

Mostly, it is not necessary to add anchors to heading 4. Do you find any link points to this heading?

Reply:
yeah, I get it, thanks.